bin/update

```diff
@@ -16,7 +16,10 @@ end
 
 Dir.chdir APP_ROOT do
   puts "== Installing dependencies =="
-  execute "bundle update"
+  execute('gem install bundler --conservative')
+
+  # Swallow non-zero bundle check exit codes and run install
+  system('bundle check') || execute('bundle install')
```

Comment:
@NickLaMuro I think that's solved with the mentality that you should run `bin/update` on each pull, and `bin/reset` on a "few times per week" basis.

My problem is I ran `bin/update` several times yesterday as I rebased several PRs at different times in the day and had to wait nearly 2 minutes to do anything.